naming/src/main/java/com/alibaba/nacos/naming/core/v2/event/service/ServiceEvent.java

```diff
@@ -56,4 +56,16 @@ public ServiceChangedEvent(Service service, boolean incrementRevision) {
             }
         }
     }
+    
+    /**
+     * Service data changed event.
+     */
+    public static class ServiceRemovedEvent extends ServiceEvent {
```

Comment:
I think this event is not useful. 

It can be replaced by ServiceMetadataEvent(service, true)